### PR TITLE
Fix chat-with-pdf doc and code issues

### PR DESCRIPTION
**Pull request by ttthree**
# Description

A few fixes:
- main.py doesn't actually take url input
- configuration of .env was mentioned after run ```python main.py```
- loading dotenv doesn't override existing env vars
- .env.example was put in wrong folder
- using gpt-4 in config input to avoid the discrepancy between oai and aoai (user doesn't need to change the model/deployment name)

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/flows/chat/chat-with-pdf/config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (wangchao1230) [inline examples/flows/chat/chat-with-pdf/config.json:1]: pls avoid check-in such config.json

File: examples/flows/chat/chat-with-pdf/config.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    "subscription_id": "d128f140-94e6-4175-87a7-954b9d27db16",
```
Comment (wangchao1230) [inline examples/flows/chat/chat-with-pdf/config.json:2]: may add config.json to gitignore file in repo root directory

Comment (ttthree) [reply examples/flows/chat/chat-with-pdf/config.json]: done.

File: examples/flows/chat/chat-with-pdf/config.json (hunk 1 above)
Comment (ttthree) [reply examples/flows/chat/chat-with-pdf/config.json]: done.

File: examples/tutorials/e2e-development/chat-with-pdf.md [hunk 3]
```diff
@@ -310,7 +326,7 @@ the other choices, please refer to [flow deploy docs](https://microsoft.github.i
 Use the command below to build a flow as docker format app:
 
 ```bash
-pf flow build --source . --output dist --format docker
+pf flow build --source . --output build --format docker
```
Comment (wangchao1230) [inline examples/tutorials/e2e-development/chat-with-pdf.md:329]: I think this should be a resolve conflict issue? 
we intentionally changed the output dir from build to dist in another PR.

**Review by wangchao1230 — APPROVED** [review]

**Review by eniac871 — APPROVED** [review]

**Pull request merged**